### ISSUE TITLE
suggest add Api Friendly evaluate

### ISSUE THREAD

**Issue by sgf** [danielpalme/IocPerformance#151]
suggest add Api Friendly evaluate,

For example, DryIoc, its API is very unfriendly and inconvenient to use.

**Comment by danielpalme** [comment]
Good idea, but I don't put any effort in this project any more.

**Comment by dadhi** [comment]
@sgf Hi, if you see something unfriendly in DryIoc api could you suggest what is it in DryIoc issue or discussion?
Better with comparison of what you think a friendly api.

@danielpalme Thanks for all the fish! :-)

**Comment by sgf** [comment]
> @sgf Hi, if you see something unfriendly in DryIoc api could you suggest what is it in DryIoc issue or discussion? Better with comparison of what you think is a friendly api.
> 
> @danielpalme Thanks for all the fish! :-)

you found this  😂

I followed the instructions from this repo and found that dryioc is very performant and functional. But I ran into various difficulties when I used it. Because the warehouse only evaluates some simple conditions, and tends to performance indicators.

dryioc lacks some default behaviors. For example: IDisposable is registered as a singleton by default to make it work.

This is just one of the problems I have, as I mentioned on your repo. Some of the problems I solved by looking up the documentation. However, I don't really want to go into too much detail. Because it's time consuming. Project The time is limited. It took me some time to learn how to use DryIOC. I really appreciate your answers in Issues, but I know that you are not obligated to do so. So in the process of learning to use DryIOC, I actually try to I don't want to disturb you.
